src/createRouter.ts

```diff
@@ -54,6 +55,8 @@ export interface RouterMixin {
 	 */
 	routes?: Route<Parameters>[];
 
+	prevHierarchy?: Selection[];
```

Comment:
Ah yea I thought I might have named it that. Struggled quite a bit with the naming.